scanner/src/routes/scan/+page.svelte

```diff
@@ -135,6 +147,17 @@
       onClick={() => showMetaData(false)}
     />
   </div>
+{:else}
```

Comment:
```suggestion
{:else if status === Status.SCAN_FAILED}
```
It's better to be explicit for the future developers who will look for this section